StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentOption+Images.swift

```diff
@@ -168,6 +168,8 @@ extension STPPaymentMethodType {
                     return .pm_type_alipay
                 case .OXXO:
                     return .pm_type_oxxo
+                case .boleto:
+                    return .pm_type_boleto
```

Comment:
Don't forget to check dark mode - might need to add .boleto to `iconRequiresTinting` above

Reply:
Good catch, this icon doesn't look good in dark mode and the tinting makes it worse, I guess I need to edit the icon
No tinting:
![image](https://github.com/stripe/stripe-ios/assets/114543737/403282d4-9ca5-45e5-b336-ef3b1d7d40a2)
Tinting:
![image](https://github.com/stripe/stripe-ios/assets/114543737/a8cc4105-69fd-4e67-90f2-90cac74a6c76)

Reply:
Fixed now
![Simulator Screenshot - iPhone 14 Pro - 2023-09-14 at 09 29 50](https://github.com/stripe/stripe-ios/assets/114543737/cea0e682-3f94-4b2e-a369-25f6216343d9)